backend/src/model/storeModel.js

```diff
@@ -0,0 +1,39 @@
+const admin = require('firebase-admin');
+const seed = require("./seed")
+
+
+// Seed array, provided for the test
+
+
+module.exports = {
+    //set a geohash for query the nearbys stores
+    async initDb() {
+        for await (store of seed) {
```

Comment:
Why did you choose firestore database? In which scenario you would change firestore for a RDB?

Reply:
Firebase is a great DB when you want to begin quickly a project, it is easy to set up  and begin to use, however, when you need to use in a large db schema it is really difficult to maintain and the querys get slow and diffucult to create

Reply:
Good choice for the project!